r-package/grf/R/input_utilities.R

```diff
@@ -30,27 +30,34 @@ validate_X <- function(X, allow.na = FALSE) {
   has.missing.values
 }
 
-validate_observations <- function(V, X) {
+validate_observations <- function(V, X, binary = FALSE) {
   if (is.matrix(V) && ncol(V) == 1) {
     V <- as.vector(V)
   } else if (!is.vector(V)) {
-    stop(paste("Observations (W, Y, or Z) must be vectors."))
+    stop(paste("Observations (W, Y, Z or C) must be vectors."))
```

Comment:
Instead of `C`, I think we should have `delta`. One aspect I liked about the old naming scheme is that observation vectors are always single uppercase characters -- would calling this `D` instead of `delta` make sense at all?

Reply:
I renamed it to `D`, it was `C` before, but Yifan wanted to use `delta`.

Reply:
And yes, Yifan has tried `surival_forest` out and OK'd it.

Reply:
(agree that using a single letter is better for consistency)